internal/report/report_test.go

```diff
@@ -213,6 +238,41 @@ var testProfile = &profile.Profile{
 	Mapping:  testM,
 }
 
+var testProfile2 = &profile.Profile{
```

Comment:
I think it would be easier to massage testProfile where needed in the test to produce this profile. As far as I can tell, it's just negating the value in one of the samples.

Reply:
Done.
I negated all samples for simplicity, but this works.